.github/workflows/python.yml

```diff
@@ -2,10 +2,6 @@ name: CI (Python)
 
 on:
   pull_request:
-    paths:
```

Comment:
We don't want to restrict wheel-building to only when we have changed python code.
1) We want to catch if if we break the wheel by changing rust code
2) We want to be able to build new wheels based on changes to the rust code